plymouth/qubes-dark/qubes-dark.script

```diff
@@ -41,6 +49,9 @@ fun dialog_setup(text)
     entry.sprite.SetX(prompt.sprite.GetX() + (prompt.image.GetWidth() - entry.image.GetWidth()) / 2);
     entry.sprite.SetY(prompt.sprite.GetY() + prompt.image.GetHeight() + 16);
     entry.sprite.SetZ(1);
+
+    help.image = Image.Text("F1 - Hide password bullets\nF2 - Show password as bullets\nF3 - Show password as text", 0.8, 0.8, 0.8);
```

Comment:
Please make it some further key, for example F8. Otherwise it's too easy to show the password when in fact you want to hide it.

Reply:
Yes, keep F1/F2. The point is to have them further apart.